querqy-core/src/test/java/querqy/rewrite/commonrules/LineParserTest.java

```diff
@@ -222,6 +222,27 @@ public void testThatPlayholdersAreParsedForBoostInstruction() {
         assertTrue(instruction instanceof BoostInstruction);
         assertTrue(((BoostInstruction) instruction).hasPlaceHolderInBoostQuery());
     }
+    
```

Comment:
Could you please add tests for malformed or invalid boosts? Also for () on the right-hand side of the rule?

Reply:
Added tests ✅